.github/workflows/update-pinned-reqs.yml

```diff
@@ -46,11 +46,12 @@ jobs:
     - name: Create pull request
       uses: peter-evans/create-pull-request@v7
       with:
-        token: ${{ steps.generate_token.outputs.token }}
-        commit-message: Update requirements with 'nox -s requirements'
         title: Update pinned requirements 📍
+        add-paths: uv.lock
+        commit-message: Update requirements with 'nox -s requirements'
+        # The PR body message is generated dynamically via `nox -s requirements` in CI
```

Comment:
This comment is the only new line.  Most of the other lines are just moved around.